mysql/datadog_checks/mysql/statement_samples.py

```diff
@@ -684,6 +692,10 @@ def _explain_statement(self, cursor, statement, schema, obfuscated_statement):
                 raise
             if e.args[0] in PYMYSQL_NON_RETRYABLE_ERRORS:
                 self._collection_strategy_cache[strategy_cache_key] = explain_strategy_error
+            self._no_explain_reason = (
+                'Unable to explain statement because the schema could not be accessed. This could be'
+                + ' the result of a missing schema, invalid permissions, or incorrect definition.'
+            )
```

Comment:
I think it would be much better to use codes here rather than text. Then we can modify the copy of the text in the UI to say something nice for that code. i.e. have a mapping like:

```
{
  "err_schema_inaccessible": 'Unable to explain statement because the schema could not be accessed. This could be the result of a missing schema, invalid permissions, or incorrect definition.',
  ...
} 

```